src/cmd/root.go

```diff
@@ -118,7 +119,7 @@ func setupLogging() {
 }
 
 func setupOutput() {
-	outputFormat := strings.ToLower(viper.GetString("output"))
+	outputFormat = strings.ToLower(viper.GetString("output"))
```

Comment:
`outputFormat` is defined on line 29. I made a safe bet and assumed this intends to update the value of that string. The `:` was creating a new, locally scoped `outputFormat` in `setupOutput()` that was having no effect outside this function call.

Reply:
oh fascinating.  I'm not sure how much this is used by customers.  But if so you likely fixed a bug. 🎉